V/VLFeat/build_tarballs.jl

```diff
@@ -0,0 +1,81 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "VLFeat"
+version = v"0.9.21"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/vlfeat/vlfeat.git", "2f6abbf13fe7ee052bc970480efaca9d6b0c195d"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd vlfeat
+
+atomic_patch -p1 ../patches/vlfeat-march.patch
+atomic_patch -p1 ../patches/vlfeat-glibc-limitation.patch
+atomic_patch -p1 ../patches/vlfeat-mingw-lowercase-windows-include.patch
+atomic_patch -p1 ../patches/vlfeat-windows.patch
+
+make_args=""
+
+if [[ $target != x86_64-* ]]; then
+    make_args+=" \
+        DISABLE_AVX=1 \
+        DISABLE_SSE2=1"
+fi
+
+if [[ $target == aarch64-apple-darwin*
```

Comment:
Simply use `*-apple-darwin*` to match both systems?

Reply:
Yeah, I was trying to be explicit about the arch's as it was in order to set VLFeat's `ARCH` property - and to avoid future accidental true if clauses (when Apple switch to their next arch. :-) ), but... yeah.

Modified as you suggested.